src/popup/router/index.ts

```diff
@@ -149,12 +149,16 @@ if (IS_MOBILE_APP) {
 
       try {
         const path = `/${event.url.slice(prefix?.length).split('?')[0]}`;
-        const query = event.url.slice(prefix?.length).split('?')[1].split('&').reduce((acc, param) => {
-          const [key, value] = param.split('=');
-          return { ...acc, [key]: value };
-        }, {});
-
-        router.push({ path, query });
+        if (path.startsWith('/invite')) {
+          router.push({ path, hash: path.split('#')[1] });
+        } else {
+          const query = event.url.slice(prefix?.length).split('?')[1].split('&').reduce((acc, param) => {
```

Comment:
This is the 2nd time where similar logic is used. I suggest to use: 

```
const [path, query] = ...split('?');
```

Or use methods available for `Url` class when instantiated on the path.